manager/cve_handler.py

```diff
@@ -271,7 +271,7 @@ def _get_cve_details(synopsis, args):
                 "rules": rules_map.get(data["id"], []),
                 "secondary_url": data["secondary_url"],
                 "synopsis": data["synopsis"],
-                "known_exploit": data["exploit_data"] is not None
+                "known_exploit": data["exploit_data"] is not None and data.get("advisories_list") is not None
```

Comment:
```suggestion
                "known_exploit": data["exploit_data"] is not None and data.get("advisories_list")
```
I suppose we want to display `false` even if `data["advisories_list"] = []`

The question is whether it is enough to check against the static list of advisories for the CVE (it contains list of advisories associated with the CVE) or if we need to check whether at least one of these advisories is applicable in a user's workload. I hope the former is enough since it is the API for displaying general CVE details.

Reply:
It was just CI test, nothing to worry about yet. I will mark PR as ready for review when I am done.

Thanks for ideas though, I will keep it in mind.